src/components/utilities/Input.js

```diff
@@ -162,6 +136,26 @@ class Input extends Component {
           readOnly={this.props.readOnly}
         />
         {
+          this.props.type === 'password' && (
+            <div
+              style={{
+                float: 'right',
+                marginTop: '-43px',
+                marginRight: '10px',
+                opacity: '0.5',
+              }}
```

Comment:
Also FYI, you can express them without unit and as integers. That would be `px` by default for sizes.